lib/taste_tester/tunnel.rb

```diff
@@ -49,9 +49,10 @@ def cmd
       else
         pid = '\\$\\$'
       end
+      @max_ping = @delta_secs / 10
       cmds = "ps -p #{pid} -o pgid | grep -v PGID" +
```

Comment:
The `ps -p #{pid} -o pgid` command never did retrieve the ssh process for me. Instead, this will retrieve the bash process that ran the command via ssh. Unfortunately, when it comes time to use this pid to kill the tunnel, it only kills the bash process and not the parent ssh tunnel. Instead of this, I used this line (courtesy of http://superuser.com/q/150117):
`cmds = "cat /proc/#{pid}/status | grep PPid | cut -f2 " +`